stdlib/public/core/Random.swift

```diff
@@ -118,8 +118,19 @@ extension RandomNumberGenerator {
 ///
 /// `Random.default` is safe to use in multiple threads, and uses a
 /// cryptographically secure algorithm whenever possible.
+///
+/// Platform Implementation of `Random`
+/// ===================================
+///
+/// - `Darwin` devices use `arc4random_buf(3)` for macOS 10.12 or higher, iOS 10
+///   or higher, tvOS 10 or higher, and watchOS 3 or higher. For older versions
+///   of these operating systems, they read from `/dev/urandom`.
```

Comment:
Could this be "Apple platforms" instead of "Darwin devices"? For example:

> Apple platforms use `arc4random_buf` (with the NIST-approved AES cipher, but not the RC4 algorithm) or `/dev/urandom`.

Reply:
How does this sound, "Apple platforms use `arc4random_buf(3)` for newer versions of their OS, and for older versions they will read from `/dev/urandom`."?

Reply:
Sounds good. Do you need to mention any others ([FreeBSD, Haiku, PS4](https://github.com/apple/swift/blob/master/lib/Basic/LangOptions.cpp)) or can they be included with the `getrandom` description?